src/jit/compiler.h

```diff
@@ -1411,8 +1411,8 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
     var_types                       GetHfaType(GenTreePtr tree);
     unsigned                        GetHfaSlots(GenTreePtr tree);
 
-    inline var_types                GetHfaType(CORINFO_CLASS_HANDLE hClass);
-    inline unsigned                 GetHfaSlots(CORINFO_CLASS_HANDLE hClass);
+    var_types                       GetHfaType(CORINFO_CLASS_HANDLE hClass);
```

Comment:
Why have you removed the inline? The member GetHfaClassHandle above still has it, so I wonder why was it needed here.